### DISCUSSION TITLE
Put car in someones garage

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#2307] [Q&A]
I saw a video where someone spawned a car and put it on someones garage.
Now i'm asking if it's currently possible to do so, and if it's possible to do with a lua script.
Thanks.

**Comment by ghost** [comment]
Gifting cars is a thing with other menus yeah. It’s possible.

**Reply by pierrelasse** [reply]
Do you have a slight idea of how it‘s done?

**Reply by ghost** [reply]
no

**Reply by Rabtos** [reply]
It's possible with Stand. But there are 3 Rules:
1. Has to be an normal garage/apartment, no CEO Garage, 50 Car Garage or Club Garage etc.
2. Garage has to be full and he has to replace the new car with an old one.
3. The target player is not allowed to have his personal vehicle outside.

Maybe this helps.

**Reply by Tessio** [reply]
2 options 

With using decorators natives 

or

Spamming a stored car when entering a 10, 6, or 2 vacant garage with one from the street.

**Reply by pierrelasse** [reply]
How would i do this with natives?
Btw thanks for the helpful responses everyone.

**Reply by YimMenuTheBest** [reply]
I used this #2262 in the previous version of GTA, it worked perfect, but with this new update it doesn't work, if you manage to find some complementation let me know.

**Reply by pierrelasse** [reply]
So i will have to wait till decors work again?

**Reply by GreySideMM** [reply]
What's the state of this?
Someone in #2680 mentioned a Lua script that can do this - after disabling ENTITY::SET_ENTITY_AS_NO_LONGER_NEEDED() ~within the Lua binding.~ (Edit: Actually depending on the way you spawn the vehicle.) See also the linked script.
Otherwise somewhere here (I couldn't find it again) mentioned a interesting method that might work here also:
When you drive a car into your garage (needs to be rather normal garage) you can replace this car: Just spawn (and sit inside) a new car. The previous car can be a random NPC car - and needs to have no insurance, otherwise it gets reset after a game restart.
